### ISSUE TITLE
Full Dataset and SFT Code Release

### ISSUE THREAD

**Issue by Yuepeng-Fu** [THUDM/Android-Lab#5]
The authors did great work on AndroidLab! I am very interested in reproducing the amazing work you have done.
According to the paper, authors constructed a tremendous dataset(726 traces, 6208 steps), and some portion of the dataset is used for fine-tuning. 
Could you please share the code and the training set or the code and the full dataset? This will be very helpful to the open-source community🤗

**Comment by xuyifan-0731** [comment]
Thank you very much for your interest in our work!

Regarding Finetuning Code:
For LLMs (qwen2, chatglm4, and llama3.1) and qwen2-vl, we used full-scale finetuning via llama-factory for training.
For cogvlm and llama3.2-vision, we conducted full-scale finetuning using ms-swift.

All training was performed with a learning rate of 1e-5, training for 3 epochs, and a batch size of 128.

Regarding Data Release:
We have also open-sourced the Android Instruct dataset mentioned in the paper. Please refer to docs/instruction_tuning.md for more details.
